index.html

```diff
@@ -1,23 +1,81 @@
 <!DOCTYPE html>
 <html>
-	<body>
-		<h1>My First Heading</h1>
-		<p>My first paragraph.</p>
 
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb"
+	 crossorigin="anonymous">
+</head>
+
+<body>
+	<div class="container">
 		<div id="target"></div>
+		<div class="row">
+			<div class="col">
+				<div id="target"></div>
+			</div>
+		</div>
+	</div>
 
-		<script src="https://cdn.jsdelivr.net/npm/ractive"></script>
+	<script src="https://cdn.jsdelivr.net/npm/ractive"></script>
 
-		<script id="template" type="text/ractive">
-			<p>{{greeting}}, {{name}}!</p>
-		</script>
+	<script id="template" type="text/ractive">
+			<h1>{{title}}</h1>
+			<h5>{{subtitle}}</h5>
+			<button on-click="@global.alert( 'Activating!' ),
+			console.log( 'alert was acknowledged' )">{{greeting}} {{name}}click to activate</button>
 
-		<script>
-			var ractive = new Ractive({
-				target: '#target',
-				template: '#template',
-				data: { greeting: 'Goodbye', name: 'Earth' }
-			});
+			<table class="table">
+					<tr>
+						<th>Name</th>
+						<th>Age</th>
+						<th>Is Minor</th>
+						<th>Action</th>
+					</tr>
+					{{#each people}}
+					<tr>
+							<td>{{name}}</td>
+							<td>{{age}}</td>
+							<td>{{isMinor}}</td>
```

Comment:
Current bug would be when age is changed the isMinor will not be available.  Consider changing it to {{age < 21}} or altering the edit so that it will change the value.